src/move/econia/doc/market.md

```diff
@@ -11,9 +11,10 @@ order book entry is added under the resource account at a new market
 ID.
 
 Once a market is registered, signing users and delegated custodians
-can place limit orders on the book as makers, takers can place
-market orders or swaps against the order book, and makers can cancel
-or change the size of any outstanding orders they have on the book.
+can place limit orders on the book as makers and/or as takers,
+takers can place market orders or swaps against the order book, and
+makers can cancel or change the size of any outstanding orders they
+have on the book.
```

Comment:
This wording seems strange to me. There is no sense in which any given user is a maker or a taker—any given order can be a maker order or a taker order, and the user will be a maker or a taker with regard to that specific order, but that doesn't describe their relationship to the market itself.

Reply:
Tagging #321 to incorporate refactors on that iteration